vignettes/xcms.Rmd

```diff
@@ -431,6 +431,56 @@ numeric).
 chromPeakData(faahko)
 ```
 
+### Chromatographic peak quality
+
+Based on the publication by Kumler et al. published in 2023 [@Kumler2023], new
+quality metrics (*beta_cor* and *beta_snr*) were added to *xcms*. *beta_cor*
+indicates how bell-shaped the chromatographic peak is and *beta_snr* is
+estimating the signal-to-noise ratio using the residuals from this fit. These
+metrics can be calculated during peak picking by setting `verboseBetaColumns =
+TRUE` in the `CentWaveParam` object, or they can be calculated afterwards by
+using the `chromPeakSummary()` function with the `XcmsExperiment` object and
+the `BetaDistributionParam` parameter object as input:
+
+```{r peak-detection-chromPeakSummary}
+beta_metrics <- chromPeakSummary(faahko, BetaDistributionParam())
+head(beta_metrics)
+
+```
+
+The result returned by `chromPeakSummary()` is thus a numeric matrix with the
+values for these quality estimates, one row for each chromatographic peak.
+Using summary statistics, one can explore the distribution of these metrics in
+the data.
+
+```{r beta-metrics}
+summary(beta_metrics)
+```
+
+Visual inspection gives a better idea of what these metrics represent in terms
+of peak quality in the data at hand. This information can be used to e.g.
+filter out peaks that don't meet a chosen quality metric threshold. In order to
+plot the detected peaks, their EIC can be extracted with the function
+`chromPeakChromatograms`. An example of a peak with a high *beta_cor* and for
+a peak  with a low *beta_cor* score is given below.
+
+```{r chromPeakChromatograms, message=FALSE}
+beta_metrics[c(4, 6), ]
+eics <- chromPeakChromatograms(
+    faahko, peaks = rownames(chromPeaks(faahko))[c(4, 6)])
+```
+
+```{r peak-quality-metrics, fig.widht = 10, fig.height = 5, fig.cap = "Plots of high and low quality peaks. Left: peak CP0004 with a beta_cor = 0.98, right: peak CP0006 with a beta_cor = 0.13."}
```

Comment:
fig.width instead of fig. widht? Not sure if this matters if the vignette still compiles nicely.